cli/lib/errors.js

```diff
@@ -191,6 +192,20 @@ const invalidCypressEnv = {
   exitCode: 11,
 }
 
+/**
+ * This error happens when CLI detects that the child Test Runner process
+ * was killed with a signal, like SIGBUS
+ * @see https://github.com/cypress-io/cypress/issues/5808
+ * @param {'close'|'event'} eventName Child close event name
+ * @param {string} signal Signal that closed the child process, like "SIGBUS"
+*/
+const childProcessKilled = (eventName, signal) => {
+  return {
+    description: `The Test Runner received event ${chalk.cyan(eventName)} with signal ${chalk.cyan(signal)}`,
```

Comment:
make it clearer that this is an error? right now it's unclear that this is an exception

```suggestion
    description: `The Test Runner unexpectedly exited via a ${chalk.cyan(eventName)} event with signal ${chalk.cyan(signal)}`,
```